src/kayenta/service/canaryRun.service.ts

```diff
@@ -22,7 +20,7 @@ export const getCanaryRun = (configId: string, canaryExecutionId: string): Promi
       const { config } = run;
       config.id = configId;
       run.id = canaryExecutionId;
-      run.result.judgeResult.results = sortBy(run.result.judgeResult.results, 'name');
+      run.result?.judgeResult.results.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Will the judgeResult always be there? Or should it be ?. as well?

Reply:
As far as I can tell, if there is a `result` field, the `judgeResult` field should also be populated (based on [this code in Kayenta](https://github.com/spinnaker/kayenta/blob/master/kayenta-core/src/main/java/com/netflix/kayenta/canary/ExecutionMapper.java#L184)).